atlp-rwanda/e-comm-team-brogrammers-fn
# 184948159 reset password

#### What does this PR do?
User can Reset password
#### Description of Task to be completed?
The user will be able to reset his password in case their forgot it
#### How should this be manually tested?
*    After cloning the repo `run npm install`
*    Then run `npm start`
*    To switch to the reset password page in your browser in the URL write `http://localhost:3000/reset-pass`
*    Then you can reset your password , you will get success toast message to check your email if the entered email is in the DB or get failed message if not found

#### Any background context you want to provide?
n/a

#### What are the relevant pivotal tracker/Trello stories?
https://www.pivotaltracker.com/n/projects/2639400/stories/184948159

#### Screenshots (if appropriate)
<img width="1440" alt="Screenshot 2023-04-24 at 18 53 21" src="https://user-images.githubusercontent.com/118971204/234064145-0d03dc1c-88c3-4207-9461-027ddd7095ab.png">


#### Questions:
